besu/src/main/java/org/hyperledger/besu/controller/MergeBesuControllerBuilder.java

```diff
@@ -69,9 +70,8 @@ protected MiningCoordinator createMiningCoordinator(
             metricsSystem,
             ethProtocolManager.ethContext(),
             syncState,
-            new BackwardSyncLookupService(
-                protocolSchedule, ethProtocolManager.ethContext(), metricsSystem, protocolContext),
-            storageProvider));
+            BackwardChain.from(
+                storageProvider, ScheduleBasedBlockHeaderFunctions.create(protocolSchedule))));
```

Comment:
I think this is going to be a problem/conflict with the reorg below TTD fix.

I neglected to merge #3696 into main, I was wanting to give it ample time for review. We need to account for using a TransitionProtocolSchedule or we are going to have problems when backwards sync leads us to a new TTD block.

I will check to see if we can resolve the conflict in a clean way.

Reply:
This was a false alarm.  The hash function that this is being used for doesn't change pre/post merge.